packages/mdctl-cli/mdctl.js

```diff
@@ -278,7 +278,11 @@ module.exports = class MdCtlCli {
 
     if (rString(this.args('file'))) {
       const file = await loadJsonOrYaml(this.args('file'))
-      Object.assign(options, _.pick(file, 'type', 'endpoint', 'env', 'username', 'apiKey'))
+      if (Array.isArray(file)) {
+        Object.assign(options, { credList: file })
+      } else {
+        Object.assign(options, _.pick(file, 'type', 'endpoint', 'env', 'username', 'apiKey'))
```

Comment:
could just add password and token to this list. it allows for less secure operation but for CI would be okay. Also, we could add env var sources for these, too, instead of requiring devs to use the file option. something like MDCTL_CLI_ARG_{ARGUMENT}?